weblate/trans/forms.py

```diff
@@ -2623,6 +2624,7 @@ class BulkEditForm(forms.Form):
     def __init__(self, user: User | None, obj, *args, **kwargs) -> None:
         project = kwargs.pop("project", None)
         kwargs["auto_id"] = "id_bulk_%s"
+        kwargs["initial"] = {"path": obj.full_slug}
```

Comment:
`obj` can be `None` here (for site-wide) and it then fails: `AttributeError: 'NoneType' object has no attribute 'full_slug'`

Reply:
Passing blank path and not using it in lookup should do the trick in that case.